src/components/NestedEditComponent.tsx

```diff
@@ -149,9 +150,10 @@ export class NestedEditComponent extends React.Component<InlineComponentProps, a
                         marginRight: "10px"
                     }}>{field.title.toUpperCase()}</label><br /></span>}
                     <textarea
+                        data-value={field.field}
                         disabled={field.readonly === true || this.props.readonly}
                         value={this.props.currentModel ? this.props.currentModel[field.field] : ""}
-                        onChange={this.handleChange.bind(this, field)}
+                        onChange={this.handleFieldChange}
                         style={{ width: 250 }} />
```

Comment:
Removed .bind(this) Functionality for textarea, input components. Added New Function (handleFieldChange) to support that functionality